tests/schemas/invalid/test_invalid_answer_source_reference.json

```diff
@@ -0,0 +1,1487 @@
+                  },
+                  "when": {
+                    "==": [
+                      {
+                        "source": "answers",
+                        "identifier": "proxy-answer"
+                      },
+                      "No, I’m answering for myself"
+                    ]
+                  }
+                },
+                {
+                  "question": {
+                    "answers": [
+                      {
+                        "id": "confirm-sex-answer",
+                        "mandatory": true,
+                        "options": [
+                          {
+                            "label": "Yes, {person_name_possessive} sex is {sex-answer}",
+                            "value": "Yes, {person_name_possessive} sex is {sex-answer}"
+                          },
+                          {
+                            "label": "No, that is not their sex",
+                            "value": "No, that is not their sex"
+                          }
+                        ],
+                        "type": "Radio"
+                      }
+                    ],
+                    "id": "confirm-sex-question",
+                    "title": {
+                      "placeholders": [
+                        {
+                          "placeholder": "person_name_possessive",
+                          "transforms": [
+                            {
+                              "arguments": {
+                                "delimiter": " ",
+                                "list_to_concatenate": [
+                                  {
+                                    "source": "answers",
+                                    "identifier": "first-name"
+                                  },
+                                  {
+                                    "source": "answers",
+                                    "identifier": "last-name"
+                                  }
+                                ]
+                              },
+                              "transform": "concatenate_list"
+                            },
+                            {
+                              "arguments": {
+                                "string_to_format": {
+                                  "source": "previous_transform"
+                                }
+                              },
+                              "transform": "format_possessive"
+                            }
+                          ]
+                        },
+                        {
+                          "placeholder": "sex-answer",
+                          "transforms": [
+                            {
+                              "arguments": {
+                                "source": "answers",
+                                "identifier": "sex-answer"
+                              },
+                              "transform": "copy"
```

Comment:
I don't think this is a valid transform? Also I'm not sure why the new question we added is also a variant, as the whole point was to remove the confusing duplication from the test.

Reply:
I've removed transform altogether as I couldn't find any suitable for this situation so only "bare" placeholder left here.